modules/navier_stokes/test/tests/finite_volume/ins/solidification/pipe_solidification.i

```diff
@@ -0,0 +1,400 @@
+mu = 8.8871e-4
+rho_solid = 997.561
+rho_liquid = 997.561
+k_solid = 0.6203
+k_liquid = 0.6203
+cp_solid = 4181.72
+cp_liquid = 4181.72
```

Comment:
This is a test for testing the mushy zone resistance. Different material properties are tested in `solidification_no_advection.i` Prefer to keep test simple so that if they fail we know what happened